specs/swagger.yml

```diff
@@ -1911,7 +1973,7 @@ paths:
           required: true
```

Comment:
Update description to "A service edge router policy update object"? Same for next one down?

Reply:
Fixed